src/H5back_gen.py

```diff
@@ -0,0 +1,472 @@
+import json
+import textwrap
+
+with open('../share/dbtypes.json') as f:
+    rawtable = json.load(f)
+
+#table = set(tuple(row[:-3]) for row in rawtable[1:])
+v3_table = set(tuple(row) for row in rawtable[897:])
```

Comment:
PEP8: global (module-level scope) variables should be named with `ALL_UPPERCASE` convention.